src/components/Popup.js

```diff
@@ -53,4 +53,21 @@ Description: closes pop up/ modal when clicked outside form
 			this._handleCloseMouseClick
 		);
 	} //end func
+
+	/**
+	 * Params: isLoading, loadingText
+	 * Description:	isLoading is a boolean value used to change the text of the submit button when calls
+	 * 						made to/from Api
+	 * 						loadingText is a string that lets the user know that request is being worked on
+	 * 						in the back
+	 */
+	renderLoading(isLoading, loadingText = "Saving...") {
```

Comment:
`renderLoading` should not be in `Popup` because `PopupWithImage` is a child of `Popup` and it doesn't have any submit button

Reply:
I think its a better idea to keep the ``renderLoading`` function in the Popup class because if its in the ``PopupWithForm`` the ``PopupDeleteConfirm`` class will have to have a separate ``renderLoading`` function since the ``PopupDeleteConfirm`` class is a child of the ``Popup`` class and not ``PopupwithForm`` class